src/main/java/frc/robot/subsystems/boathook/Boathook.java

```diff
@@ -68,68 +75,84 @@ public class Boathook extends SubsystemBase {
                   .withFeedbackSensorSource(FeedbackSensorSourceValue.RemoteCANcoder)
                   .withRotorToSensorRatio(25 * (34.0 / 24.0))
-                  .withSensorToMechanismRatio(1.8))
+                  .withSensorToMechanismRatio(1.8)
+                  .withFeedbackRotorOffset(0))
           .withSoftwareLimitSwitch(
               new SoftwareLimitSwitchConfigs()
                   .withForwardSoftLimitEnable(true)
                   .withForwardSoftLimitThreshold(BoathookConstants.EXTENDER_FORWARD_LIMIT)
-                  .withReverseSoftLimitEnable(true)
-                  .withReverseSoftLimitThreshold(BoathookConstants.EXTENDER_REVERSE_LIMIT))
+                  .withReverseSoftLimitEnable(false))
           //          .withMotionMagic(
           //              new MotionMagicConfigs()
           //                  .withMotionMagicAcceleration(2)
           //                  .withMotionMagicCruiseVelocity(1))
-          .withSlot0(new Slot0Configs().withKP(32).withKD(0).withKG(0))
+          .withSlot0(new Slot0Configs().withKP(20).withKD(0).withKG(0))
           .withMotorOutput(new MotorOutputConfigs().withInverted(InvertedValue.Clockwise_Positive))
-          .withSoftwareLimitSwitch(
-              new SoftwareLimitSwitchConfigs()
-                  .withReverseSoftLimitEnable(true)
-                  .withReverseSoftLimitThreshold(-0.15)
-                  .withForwardSoftLimitEnable(true)
-                  .withForwardSoftLimitThreshold(4.5));
-  // .withHardwareLimitSwitch(
-  //     new HardwareLimitSwitchConfigs()
-  //         .withReverseLimitEnable(false)
-  //         .withReverseLimitAutosetPositionEnable(false)
-  //         .withReverseLimitAutosetPositionValue(4)
-  //         .withReverseLimitRemoteSensorID(BoathookConstants.BOATHOOK_CANDI_ID)
-  //         .withReverseLimitSource(ReverseLimitSourceValue.RemoteCANdiS2)
-  //         .withReverseLimitType(ReverseLimitTypeValue.NormallyClosed));
-
-  CANcoderConfiguration extentionCANcoderConfig =
+          .withHardwareLimitSwitch(
+              new HardwareLimitSwitchConfigs()
+                  .withReverseLimitEnable(true)
+                  .withReverseLimitAutosetPositionEnable(true)
+                  .withReverseLimitAutosetPositionValue(BoathookConstants.EXTENDER_REVERSE_LIMIT)
+                  .withReverseLimitRemoteSensorID(BoathookConstants.CANDI_ID)
+                  .withReverseLimitSource(ReverseLimitSourceValue.RemoteCANdiS2)
+                  .withReverseLimitType(ReverseLimitTypeValue.NormallyOpen)
+                  .withForwardLimitEnable(false)
+                  .withForwardLimitAutosetPositionEnable(false));
+
+  CANcoderConfiguration extensionEncoderConfig =
       new CANcoderConfiguration()
           .withMagnetSensor(
               new MagnetSensorConfigs()
-                  .withMagnetOffset(0.6)
+                  .withMagnetOffset(0) // 0.372
```

Comment:
This looks like it was reset for testing with a comment to suggest its initial value. Actually I think I was the one who typed that. Something we should probably double check on the robot.